FMDB.podspec

```diff
@@ -26,24 +31,28 @@ Pod::Spec.new do |s|
   s.subspec 'standalone' do |ss|
     ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
     ss.dependency 'sqlite3'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-  end
-
-  # build with FTS support and custom FTS tokenizer source files
+    ss.dependency 'FMDB/common'
+	end
+	
+  # build the latest stable version of sqlite3 
+	# with FTS support and custom FTS tokenizer source files
   s.subspec 'standalone-fts' do |ss|
```

Comment:
I like this idea and it's similar to how it was being done a long time ago in this podfile.  As above, I left the weirdness to minimize the changes I was making. Better to just fix it properly.